persistence/javadsl/src/main/java/com/lightbend/lagom/javadsl/persistence/PersistentEntityRef.java

```diff
@@ -70,16 +73,28 @@ public String entityId() {
     CompletionStage<Object> future =
         Patterns.ask(region, new CommandEnvelope(entityId, command), timeout);
 
-    return future.thenCompose(
-        result -> {
-          if (result instanceof Throwable) {
-            CompletableFuture<Reply> failed = new CompletableFuture<>();
-            failed.completeExceptionally((Throwable) result);
-            return failed;
-          } else {
-            return CompletableFuture.completedFuture((Reply) result);
-          }
-        });
+    return future
+        .thenCompose(
+            result -> {
+              if (result instanceof Throwable) {
+                CompletableFuture<Reply> failed = new CompletableFuture<>();
+                failed.completeExceptionally((Throwable) result);
+                return failed;
+              } else {
+                return CompletableFuture.completedFuture((Reply) result);
+              }
+            })
+        .whenComplete(
+            (result, throwable) -> {
+              if (throwable != null)
+                logger.error(
```

Comment:
I'd say this is a warning, not an error.
The drawback with error could be that users expect that it will never happen, which could generate more support cases (questions).

Reply:
If the message isn't retried (AFAICT) isn't an error more appropriate?

Reply:
I chose for error because that's what is already showing up in Lagom when a user gets such an exception. At the end, an exception will be logged, in error level, the `AskTimeoutException`. 

The problem is that we don't have enough information about it, we only see the shard region.

Reply:
So you mean that we will log two errors for each ask timeout now?
Couldn't we amend the exception message instead so that it's included in the other logging?

Reply:
yep, I will do so. See my comment on the akka repo discussion. That aligns better with what akka typed is doing.